packages/eslint-plugin-template/src/rules/eqeqeq.ts

```diff
@@ -0,0 +1,98 @@
+import type { AST, Binary } from '@angular/compiler';
+import { ASTWithSource, LiteralPrimitive } from '@angular/compiler';
+import {
+  createESLintRule,
+  ensureTemplateParser,
+} from '../utils/create-eslint-rule';
+import { getNearestNodeFrom } from '../utils/get-nearest-node-from';
+
+type Options = [{ readonly allowNilCheck?: boolean }];
+export type MessageIds = 'eqeqeq';
+export const RULE_NAME = 'eqeqeq';
+const DEFAULT_OPTIONS: Options[0] = { allowNilCheck: false };
+
+export default createESLintRule<Options, MessageIds>({
+  name: 'eqeqeq',
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Requires `===` and `!==` in place of `==` and `!=`',
+      category: 'Best Practices',
+      recommended: false,
+    },
+    fixable: 'code',
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          allowNilCheck: {
```

Comment:
It's confusing to use the term "nil" in the context of a language which doesn't have it IMO. Please rename the option

Reply:
Hmmm, what about: 

- `allowNullish` (ref: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/sort-type-union-intersection-members.md#options);
- `allowNullOrUndefined`;
- enter a suggestion.

Reply:
Either of those is good I think, "nullish" is a term in the language now thanks to the new operator, and null or undefined is descriptive :)

Reply:
Sorry missed your final point, I'm not sure to be honest, I don't have a great sense of which is more "idiomatic" for equivalent options in equivalent rules